microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -256,17 +272,28 @@ Microsoft Edge indicates when your precise location is being shared on the right
 
-If you turn off the **Allow access to location on this device** and **Allow apps to access your location** settings, some sites can use information such as your IP address to determine your device's imprecise location. For more information about Windows location settings, navigate to [Windows location service and privacy](https://support.microsoft.com/help/4468240).
+If you turn off the **Allow access to location on this device** and **Allow apps to access your location** settings, some sites may still estimate your location using other technologies (such as Bluetooth, WiFi, cellular modem, or IP address) with varying degrees of accuracy. With Windows location settings disabled, precise location experiences that you allow in Microsoft Edge may also be inaccurate. For more information about Windows location settings, navigate to [Windows location service and privacy](https://support.microsoft.com/help/4468240).
 
-Microsoft Edge doesn't store your geolocation coordinates. When making requests to the Microsoft location service, Microsoft Edge generates a new random ID for each request.
+Microsoft Edge doesn't store your geolocation coordinates. When making requests to the Windows location service, Microsoft Edge generates a new random ID for each request.
 
 **InPrivate** browsing uses the precise location permission setting of the profile from which the **InPrivate** session was launched. **Guest** mode always asks you for permission before granting the site your precise location.
 
 
+<!-- ====================================================================== -->
+## Image Descriptions
+When browsing the web in Microsoft Edge, screen reader users may encounter images that are not annotated with alt text. Without alt text to describe these images, they are effectively invisible to screen reader users. When Image Descriptions is turned on, Microsoft Edge will detect these images without alt text and send them to Azure Cognitive Services to generate captions. The generated captions are then announced to screen reader users. Even when Image Descriptions is turned on, images are only sent to the service when a screen reader (or other assistive technology) is connected to Microsoft Edge. 
+
+Only raw image data is sent to Azure Cognitive Services. Images are sent un-encrypted to reduce latency. No user identifiers are included in the requests to the service. Images are not stored or saved on Microsoft servers; Images are discarded immediately after the captions are generated. 
+
+To control Image Descriptions, navigate to edge://settings/accessibility. Toggle the **Get image descriptions from Microsoft for screen readers** setting. 
```

Comment:
```suggestion
To control Image Descriptions, go to `edge://settings/accessibility`, and then toggle the **Get image descriptions from Microsoft for screen readers** setting. 
```

Reply:
Every reference to an edge settings url uses the terminology "navigate to edge://settings/xxx". I'm seeing I accidently added a "go to" for geolocation. Can we keep the navigate terminology to be consistent across the document?